client/js/components/user/DpOrganisationList/DpOrganisationListItem.vue

```diff
@@ -159,6 +168,42 @@ export default {
   },
 
   methods: {
+    createAddonPayload () {
+      return {
+        type: this.addonPayload.resourceType,
+        attributes: this.addonPayload.attributes,
+        relationships: this.addonPayload.url === 'api_resource_update' ? undefined : {
+          orga: {
+            data: {
+              type: 'Orga',
+              id: this.organisation.id
+            }
+          }
+        },
+        ...(this.addonPayload.url === 'api_resource_update' ? { id: this.addonPayload.id } : {}),
+      }
+    },
+
+    handleAddonRequest () {
+      const payload = this.createAddonPayload()
+
+      const addonRequest = dpApi({
+        headers: {
+          ...(dplan.csrfToken && { 'x-csrf-token': dplan.csrfToken }) // TODO: should be adjusted in UI: api2defaultHeaders
```

Comment:
Shouldn't we remove the csrf token here, too?
Also (but not in this PR), maybe we could extract these functions into another file so we don't have to duplicate them.